src/components/menu/menu.tsx

```diff
@@ -2,43 +2,45 @@ import React, { ReactComponentElement, ReactNode, useState } from "react"
 import { MdHome, MdMail, MdMenu, MdPeople } from "react-icons/md"
 
 import styles from "./menu.module.scss"
+import { Link } from "gatsby"
 
 type MenuItem = {
   name: string
   icon: ReactComponentElement<any>
   path: string
 }
 
-export const Menu: React.FC = () => {
+export const Menu: React.FC = (props) => {
```

Comment:
do we need type here ?

Reply:
hmm we don't even need this `props` here right now cuz nothing is passing from higher component xD I will remove it for now. Thanks 👍